doc/source/conf.py

```diff
@@ -0,0 +1,67 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "nomenclature"
+copyright = "2021, IAMconsortium"
+author = "IAMconsortium"
+
+# The full version, including alpha/beta/rc tags
+release = "0.1"
```

Comment:
Please retrieve the version number automatically, see https://github.com/IAMconsortium/pyam/blob/20df347799bc0f8dc28bae9863ed71a7ae704267/doc/source/conf.py#L74